README.md

```diff
@@ -53,7 +53,7 @@ Laravel Ownership simplify management of eloquent model's owner. Group can be an
 First, pull in the package through Composer.
 
 ```sh
-$ composer require cybercog/laravel-ownership
```

Comment:
Keep it, please.

Reply:
No problem, reverting it. Change was based on rule [MD014](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md014).